pkg/template/config_context.go

```diff
@@ -297,7 +298,7 @@ func (ctx ConfigCtx) localImageName(imageRef string) (string, error) {
 		}
 		destImage, err := imageutil.DestImage(destRegistry, imageRef)
 		if err != nil {
-			// TODO: log
+			logger.Warnf("Failed to get dest image for %s: %v", imageRef, err)
```

Comment:
of course i added these and the unit test is no longer failing...

Reply:
yeah i believe these logs can make it to the cli output as this code path is hit during `kots pull`. might be fine though.

Reply:
i think it's better to have them